backend_addon/{{ cookiecutter.__folder_name }}/Makefile

```diff
@@ -92,10 +91,24 @@ console: instance/etc/zope.ini ## Start a console into a Plone instance
 create-site: instance/etc/zope.ini ## Create a new site from scratch
 	PYTHONWARNINGS=ignore $(BIN_FOLDER)/zconsole run instance/etc/zope.conf ./scripts/create_site.py
 
-.PHONY: check
-check: $(BIN_FOLDER)/tox ## Check and fix code base according to Plone standards
+# QA
+.PHONY: lint
+lint: $(BIN_FOLDER)/uv ## Check and fix code base according to Plone standards
+	@echo "$(GREEN)==> Lint codebase$(RESET)"
+	@uvx ruff@latest check --fix --config $(BACKEND_FOLDER)/pyproject.toml
+	@uvx pyroma@latest -d .
+	@uvx check-python-versions@latest .
+	@uvx zpretty@latest --check src
+
+.PHONY: format
+format: $(BIN_FOLDER)/uv ## Check and fix code base according to Plone standards
 	@echo "$(GREEN)==> Format codebase$(RESET)"
-	$(BIN_FOLDER)/tox -e lint
+	@uvx ruff@latest check --select I --fix --config $(BACKEND_FOLDER)/pyproject.toml
+	@uvx ruff@latest format --config $(BACKEND_FOLDER)/pyproject.toml
+	@uvx zpretty@latest -i src
```

Comment:
Do we want to limit the files to `--include="\.xml$"`?

Reply:
No, because I also want to check for zcml in one run